doc/sphinx/inter_non-bonded.rst

```diff
@@ -559,60 +559,78 @@ DPD interaction
     Feature ``DPD`` required.
 
 This is a special interaction that is to be used in conjunction with the
-`Dissipative Particle Dynamics (DPD)` thermostat, for a general description
-of the algorithm see there. The parameters can be set via::
+:ref:`Dissipative Particle Dynamics (DPD)` thermostat.
+The parameters can be set via::
 
     system.non_bonded_inter[type1, type2].dpd.set_params(**kwargs)
 
-This command defines a velocity-dependent interaction
-between particles of the types ``type1`` and ``type2``. For a description of the input arguments
-see :class:`espressomd.interactions.DPDInteraction`. The interaction
-only has an effect if the DPD thermostat activated, and acts according to the
-temperature of the thermostat.
+This command defines an interaction between particles of the types ``type1`` and ``type2``
+that contains velocity-dependent friction and noise according 
+to a temperature set by :py:attr:`espressomd.thermostat.Thermostat.set_dpd()`
```

Comment:
A mix of `:py:attr:` and `:py:func:` is used here and elsewhere to link to class methods. Although the HTML output looks the same, the documentation would probably be easier to maintain if we have a consistent use of `:meth:` for class methods, `:attr:` for class members and `:func:` for free functions, without the `:py:` prefix.